### PR TITLE
Starting changing parents to ancestors

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Implements https://github.com/elifesciences/api-raml/pull/211.

(Also implements https://github.com/elifesciences/api-raml/pull/209.)

### CODE REVIEW

File: tests/Annotations/Serializer/HypothesisClientAnnotationNormalizerTest.php [hunk 1]
```diff
@@ -179,7 +183,6 @@ public function normalizeProvider() : array
                         'title' => 'title',
                         'uri' => 'uri',
                     ],
-                    'parents' => [],
```
Comment (giorgiosironi) [inline tests/Annotations/Serializer/HypothesisClientAnnotationNormalizerTest.php]: to be clear, this is unrelated because the field is optional?

Comment (thewilkybarkid) [reply tests/Annotations/Serializer/HypothesisClientAnnotationNormalizerTest.php]: Yep.
